pkgs/by-name/bl/blueutil/package.nix

```diff
@@ -22,8 +19,7 @@ stdenv.mkDerivation (finalAttrs: {
   };
 
   buildInputs = [
-    Foundation
-    IOBluetooth
+    apple-sdk
```

Comment:
```suggestion
```

Reply:
don't need to add `apple-sdk` as it is part of the stdenv

Reply:
https://github.com/NixOS/nixpkgs/pull/353439